File: android/apps/AutoTesterAndroid/app/src/main/java/com/mousebirdconsulting/autotester/TestCases/MapTilerCircleTestCase.kt
package com.mousebirdconsulting.autotester.TestCases

import android.app.Activity

class MapTilerCircleTestCase(activity: Activity) :
        MapTilerTestCase(activity, "MapTiler Circles")
{
    override fun getMaps() = listOf(
        Pair("maptiler_test_circles.json", false)
    )
    
    //override fun getStyleJson(whichMap: Int) =
    //override fun setup(map: MapboxKindaMap) {
    //    super.setup(map)
    //    map.styleSettings.markerImportance = 200000.0 //Float.MAX_VALUE.toDouble()
    //}
}

File: ios/apps/AutoTester/AutoTester/testCases/MapTilerCircleTestCase.swift
//
//  MapTilerCircleTestCase.swift
//  AutoTester
//
//  Created by Tim Sylvester on 3/23/21.
//  Copyright © 2021 mousebird consulting. All rights reserved.
//
import UIKit

class MapTilerCircleTestCase: MapTilerTestCase {

    init() {
        super.init("MapTiler Circles")
        mapTilerStyle = 0
    }

    override func setup(_ map: MapboxKindaMap) {
        super.setup(map)
        map.styleSettings.markerScale = 1.0
    }

    override func getStyles() -> [(name: String, sheet: String)] {
        return [
            ("Custom", "maptiler_test_circles")
        ]
    }

}

